tests/bfd/test_bfd.py

```diff
@@ -114,15 +114,15 @@ def get_neighbors_scale(duthost, tbinfo, ipv6=False, scale_count=1):
         if ipv6:
             local_addrs.append(t1_ipv6_pattern.format(idx * 2))
             neighbor_addrs.append(t1_ipv6_pattern.format(idx * 2 + 1))
-            neighbor_devs.append(neighbor_intfs[index])
-            ptf_devs.append(ptf_intfs[index])
+            neighbor_devs.append(neighbor_intfs[index % len(neighbor_intfs)])
```

Comment:
Could you add comments for the changes?

Reply:
@kevinskwang : when used with scale_count=128, the script passes without any modifications. But any higher number runs into the index error issues. The arrays: neighbor_intfs and ptf_intfs are filled only upto 128. Beyond that we need to re-use the same addresses. My changes above basically do this by using the modulus(% operation) instead of the actual index intself.

Reply:
Could you add the comment in the PR?

Reply:
@kevinskwang : Done.